src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  padding: 10px 0;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__item {
+  list-style: none;
+  display: inline-block;
```

Comment:
No need to use inline-block here, block would be enough, u have the flex wrapper